test/external_scenario-param.sch

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema queryBinding="xslt2" xmlns:sch="http://purl.oclc.org/dsdl/schematron"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+	<xsl:param name="phase" />
+
+	<sch:phase id="A">
+		<sch:active pattern="pattern-A" />
+	</sch:phase>
+
+	<sch:pattern id="pattern-A">
+		<sch:rule context="context-child">
+			<sch:report id="compile-time-phase-param-is-A" test="true()" />
+			<sch:report id="run-time-phase-param-is-A" test="$phase eq 'A'" />
```

Comment:
I like how this schema and its test illustrate the difference between compile-time usage and run-time usage.